OpenOversight/app/templates/sort.html

```diff
@@ -13,36 +35,48 @@
       </div>
 
       <div class="row">
-        <div class="frontpage-leads">
-          <div>
-            <img class="center-block img-responsive" src="{{ path }}" alt="Picture to be sorted">
-          </div>
-        </div>
-      </div>
-
-      <div class="row">
         <div class="col-sm-4 text-center">
           <form action="{{ url_for('main.classify_submission', image_id=image.id, contains_cops=0) }}" method="post">
-            <button type="submit" name="action" class="btn btn-lg btn-danger">
+            <button type="submit" name="action" id="answer-no" class="btn btn-lg btn-danger">
               <span class="glyphicon glyphicon glyphicon-remove" aria-hidden="true"></span>
               No
             </button>
           </form>
         </div>
         <div class="col-sm-4 text-center">
-          <a href="{{ url_for('main.sort_images') }}" class="btn btn-lg btn-primary" role="button">
+          <a href="{{ url_for('main.sort_images') }}" id="answer-skip" class="btn btn-lg btn-primary" role="button">
             <span class="glyphicon glyphicon-repeat" aria-hidden="true"></span> Skip</a>
         </div>
         <div class="col-sm-4 text-center">
           <form action="{{ url_for('main.classify_submission', image_id=image.id, contains_cops=1) }}" method="post">
-            <button type="submit" name="action" class="btn btn-lg btn-success">
+            <button type="submit" name="action" id="answer-yes" class="btn btn-lg btn-success">
               <span class="glyphicon glyphicon glyphicon-ok" aria-hidden="true"></span>
               Yes!
             </button>
           </form>
         </div>
       </div>
 
+      <div class="row">
+        <div class="col-sm-4 text-center">
+          (shortcut: n)
+        </div>
+        <div class="col-sm-4 text-center">
+          (shortcut: s)
+        </div>
+        <div class="col-sm-4 text-center">
+          (shortcut: y)
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="frontpage-leads">
+          <div>
+            <img class="center-block img-responsive" src="{{ path }}" alt="Picture to be sorted">
+          </div>
+        </div>
+      </div>
+
```

Comment:
@jujueyeball This PR puts those buttons above the photo, so that part's done. Could add that tip, but that's similar to what I've implemented above, more elegant, but also a bit less clear IMO.